src/test/user.spec.js

```diff
@@ -674,3 +675,111 @@ describe('PATCH /api/v1/auth/user/:id/role', () => {
     expect(userRoleStatus.length).to.equal(1);
   });
 });
+
+describe(' POST /api/v1/auth/reset-password', async () => {
+  const newUser = await createUser();
+  const user = await BaseRepository.findAll(db.User, {
+    id: newUser.email
+  });
+  it('it should send password reset link to the email provided if it is associated with a registered email', done => {
```

Comment:
Good job @olorunwalawrence but does this user you are sending the email already exist on the database? at what point did the user get registered/inserted into the db?